src/components/MapDetail/BoothComment.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+
+const CommentTop = styled.div`
+    display: flex;
+    align-items: center;
+    padding-bottom: 1rem;
+
+    svg{
+        margin-right: 1rem;
+    }
+`;
+
+const CommentBox = styled.div`
+    padding: 3rem;
+
+    p{
+        padding-left: 34px;
+        font-weight: 400;
+    }
+
+    h3{
+        color: #969FA9;
+        font-size: 1.4rem;
+    }
+
+    div{
+        color: #BBC7D3;
+        font-size: 1.4rem;
+    }
+`;
+
+type BoothCommentProps = {
+    boothComments: Comment[];
+}
+
+export default function BoothComment({ boothComments }: BoothCommentProps) {
+  return (
+    <>
+      {boothComments.map((boothComment) => {
+        const { id, content } = boothComment;
```

Comment:
취향이긴 하지만 객체의 프라퍼티가 2개정도면 요렇게 표현해줘도 괜찮은 것 같아요~
```suggestion
      {boothComments.map(({id, content}) => {
```